### PR TITLE
Update Minio Erasure Code Quickstart Guide

### PR DESCRIPTION
**Pull request by jamesneimanconsulting**
@deekoder m @harshavardhana Kindly review and merge if approved.

### CODE REVIEW

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-tls.sh | :heavy_check_mark: |
| mint-compression-xl.sh | :heavy_check_mark: |
| mint-xl.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-compression-fs.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :x: [more...](#0b281e99-5a88-4a7f-b446-932a71444c07) |
<a name="0b281e99-5a88-4a7f-b446-932a71444c07"></a>
### 6841-59ac956/mint-large-bucket.sh.log:
```
Running with
SERVER_ENDPOINT:      minikube:31239
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 0d1faff9f1a4:/mint/log /tmp/mint-logs'

(1/13) Running awscli tests ... done in 1 minutes and 33 seconds
(2/13) Running aws-sdk-go tests ... done in 2 seconds
(3/13) Running aws-sdk-java tests ... done in 2 seconds
(4/13) Running aws-sdk-php tests ... done in 55 seconds
(5/13) Running aws-sdk-ruby tests ... done in 31 seconds
(6/13) Running mc tests ... done in 1 minutes and 29 seconds
(7/13) Running minio-dotnet tests ... FAILED in 5 minutes and 58 seconds

Unhandled Exception: System.AggregateException: One or more errors occurred. (One or more errors occurred. (Minio API responded with message=Unsuccessful response from server without XML error: The operation has timed out.) (Minio API responded with message=Unsuccessful response from server without XML error: The operation has timed out.) (Minio API responded with message=Unsuccessful response from server without XML error: The operation has timed out.) (Minio API responded with message=Unsuccessful response from server without XML error: The operation has timed out.) (Minio API responded with message=Unsuccessful response from server without XML error: The operation has timed out.)) ---> System.AggregateException: One or more errors occurred. (Minio API responded with message=Unsuccessful response from server without XML error: The operation has timed out.) (Minio API responded with message=Unsuccessful response from server without XML error: The operation has timed out.) (Minio API responded with message=Unsuccessful response from server without XML error: The operation has timed out.) (Minio API responded with message=Unsuccessful response from server without XML error: The operation has timed out.) (Minio API responded with message=Unsuccessful response from server without XML error: The operation has timed out.) ---> Minio.Exceptions.InternalClientException: Minio API responded with message=Unsuccessful response from server without XML error: The operation has timed out.
   at Minio.MinioClient.ParseError(IRestResponse response) in /q/.q/sources/minio-dotnet/Minio/MinioClient.cs:line 450
   at Minio.MinioClient.HandleIfErrorResponse(IRestResponse response, IEnumerable`1 handlers, DateTime startTime) in /q/.q/sources/minio-dotnet/Minio/MinioClient.cs:line 505
   at Minio.MinioClient.ExecuteTaskAsync(IEnumerable`1 errorHandlers, IRestRequest request, CancellationToken cancellationToken) in /q/.q/sources/minio-dotnet/Minio/MinioClient.cs:line 351
   at Minio.MinioClient.PutObjectAsync(String bucketName, String objectName, String uploadId, Int32 partNumber, Byte[] data, Dictionary`2 metaData, CancellationToken cancellationToken) in /q/.q/sources/minio-dotnet/Minio/ApiEndpoints/ObjectOperations.cs:line 483
   at Minio.MinioClient.PutObjectAsync(String bucketName, String objectName, Stream data, Int64 size, String contentType, Dictionary`2 metaData, CancellationToken cancellationToken) in /q/.q/sources/minio-dotnet/Minio/ApiEndpoints/ObjectOperations.cs:line 229
   at Minio.Functional.Tests.FunctionalTest.PutObject_Task(MinioClient minio, String bucketName, String objectName, String fileName, String contentType, Int64 size, Dictionary`2 metaData, MemoryStream mstream) in /mint/run/core/minio-dotnet/FunctionalTest.cs:line 740
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)
   at System.Threading.Tasks.Task.Wait()
   at Minio.Functional.Tests.FunctionalTest.RemoveObjects_Test2(MinioClient minio) in /mint/run/core/minio-dotnet/FunctionalTest.cs:line 1738
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)
   at System.Threading.Tasks.Task.Wait()
   at Minio.Functional.Tests.FunctionalTest.Main(String[] args) in /mint/run/core/minio-dotnet/FunctionalTest.cs:line 218

Executed 6 out of 13 tests successfully.
```

File: docs/erasure/README.md [hunk 1]
```diff
@@ -1,42 +1,71 @@
...
+This guide describes what Minio erasure code is and how to use it.
 
-Erasure code is a mathematical algorithm to reconstruct missing or corrupted data. Minio uses Reed-Solomon code to shard objects into variable data and parity blocks. For example, in a 12 drive setup, an object can be sharded to a variable number of data and parity blocks across all the drives - ranging from six data and six parity blocks to ten data and two parity blocks.
+1. [What is Erasure Code?](#whatiserasure) 
+2. [Why is Erasure Code Useful?](#whyuseful) 
+3. [What is Bit Rot Protection?](#whatisrot) 
+4. [Get Started with Minio Erasure Code](#getstarted)
 
-By default, Minio shards the objects across N/2 data and N/2 parity drives. Though, you can use [storage classes](https://github.com/minio/minio/tree/master/docs/erasure/storage-class) to use a custom configuration. We recommend N/2 data and parity blocks, as it ensures the best protection from drive failures.
+## 1. <a name="whatiserasure"></a>What is Erasure Code?
```
Comment (eco-minio) [inline docs/erasure/README.md]: Change anchor to "what-is-erasure-code"

**Review by eco-minio — CHANGES_REQUESTED** [review]

**Comment by jamesneimanconsulting** [comment]
@eco-minio Changes resolved.

**Review by eco-minio — APPROVED** [review]

File: docs/erasure/README.md [hunk 2]
```diff
@@ -1,42 +1,71 @@
...
+This guide describes what Minio erasure code is and how to use it.
 
-Erasure code is a mathematical algorithm to reconstruct missing or corrupted data. Minio uses Reed-Solomon code to shard objects into variable data and parity blocks. For example, in a 12 drive setup, an object can be sharded to a variable number of data and parity blocks across all the drives - ranging from six data and six parity blocks to ten data and two parity blocks.
+1. [What is Erasure Code?](#what-is-erasure-code) 
+2. [Why is Erasure Code Useful?](#why-is-erasure-code-useful) 
+3. [What is Bit Rot Protection?](#what-is-bit-rot-protection) 
+4. [Get Started with Minio Erasure Code](#get-started-with-minio-erasure-code)
 
-By default, Minio shards the objects across N/2 data and N/2 parity drives. Though, you can use [storage classes](https://github.com/minio/minio/tree/master/docs/erasure/storage-class) to use a custom configuration. We recommend N/2 data and parity blocks, as it ensures the best protection from drive failures.
+## 1. <a name="what-is-erasure-code"></a>What is Erasure Code?
```
Comment (deekoder) [inline docs/erasure/README.md:12]: We need to mention that customer can override this configuration using storage class. Although we recommend n/2 data & parity blocks.

File: docs/erasure/README.md [hunk 3]
```diff
@@ -1,42 +1,71 @@
...
+*Erasure code* is a mathematical algorithm to reconstruct missing or corrupted data. Minio uses Reed-Solomon code to shard objects into variable data and parity blocks. 
 
-## Why is Erasure Code useful?
-
-Erasure code protects data from multiple drives failure, unlike RAID or replication. For example, RAID6 can protect against two drive failure whereas in Minio erasure code you can lose as many as half of drives and still the data remains safe. Further, Minio's erasure code is at the object level and can heal one object at a time. For RAID, healing can be done only at the volume level which translates into high downtime. As Minio encodes each object individually, it can heal objects incrementally. Storage servers once deployed should not require drive replacement or healing for the lifetime of the server. Minio's erasure coded backend is designed for operational efficiency and takes full advantage of hardware acceleration whenever available.
+By default, Minio shards the objects across N/2 data and N/2 parity drives:
```
Comment (deekoder) [inline docs/erasure/README.md:16]: A concrete example with 12 disks was valuable.

**Review by deekoder — CHANGES_REQUESTED** [review]

**Pull request merged**